tests/test_examples.py

```diff
@@ -17,6 +17,7 @@
     'air/main.py --num-steps=1',
     'air/main.py --num-steps=1 --no-baseline',
     'baseball.py --num-samples=200 --warmup-steps=100 --num-chains=2',
+    'lkj.py --n=50 --num-chains=1 --warmup-steps=100 --num-samples=200',
```

Comment:
You'll also need to add this to `TEST_JIT` and `TEST_CUDA`, otherwise, `test_coverage` would complain that the example is not being tested. If JIT or CUDA does not work on this example, you can use `xfail` markers and specify why we need to make an exception for this example. To quickly test locally, you can also run `make test-examples` to see that all checks pass.

Reply:
Thanks; hopefully clear now.  If its not -- I'm not sure this example, which is really only intended to show the boilerplate for combining a correlation l cholesky factor and a vector of variances, is a book that's worth the candle.